src/gui/settingsDialog/uploader/ImgurUploaderSettings.cpp

```diff
@@ -152,6 +169,8 @@ void ImgurUploaderSettings::loadConfig()
 	mOpenLinkInBrowserCheckbox->setChecked(mConfig->imgurOpenLinkInBrowser());
 	mDirectLinkToImageCheckbox->setChecked(mConfig->imgurLinkDirectlyToImage());
 	mAlwaysCopyToClipboardCheckBox->setChecked(mConfig->imgurAlwaysCopyToClipboard());
+	mUploadTitleEdit->setText(mConfig->imgurUploadTitle());
```

Comment:
Maybe you could add the default value as placeholder text, then it's going to be grayed out as long as it's the default. Check out how the BaseUrl was done.